app/scripts/lib/stores/extension-store.test.ts

```diff
@@ -28,10 +28,6 @@ const setup = (
   return new ExtensionStore();
 };
 describe('ExtensionStore', () => {
-  beforeEach(() => {
```

Comment:
I found this while searching for references to `captureException`. It seems that it was an unused mock.